cirq-core/cirq/ops/dense_pauli_string.py

```diff
@@ -64,6 +60,9 @@ class BaseDensePauliString(raw_types.Gate, metaclass=abc.ABCMeta):
     Y_VAL = 2
     Z_VAL = 3
 
+    pauli_mask: np.ndarray
+    coefficient: Union[sympy.Basic, complex]
```

Comment:
Why do we need to declare class variables here?

Reply:
No need indeed. I don't actually remember adding those two... strange